core/templates/dev/head/pages/library/SearchBarDirective.js

```diff
@@ -205,6 +205,7 @@ oppia.directive('searchBar', [function() {
             }
 
             refreshSearchBarLabels();
+            $rootScope.$broadcast('searchBarLoaded', true);
```

Comment:
I noticed that sometimes on the library/search page, the custom nav elements from the SearchBarDirective would load/show up after the function had already run, causing the overflow not to be handled. Sending the event from there makes sure to run the function again late enough to work.